assets/r/model_template.R

```diff
@@ -112,6 +112,8 @@ if (<SPLIT_DATASET>) {
 
   split <- c(<DATA_SPLIT_TR_TU_TE>)
   
+  # tc is added to store the complete dataset indices for later use.
+
```

Comment:
What's the purpose of sampling the complete dataset? It will be the complete dataset again, won't it?

Reply:
Would `tc <- 1:tcnobs` also work?